code/modules/mining/machine_processing.dm

```diff
@@ -54,7 +54,8 @@
 	name = "production machine console"
 	icon = 'icons/obj/machines/mining_machines.dmi'
 	icon_state = "console"
-	density = TRUE
+	density = FALSE
+	circuit = /obj/item/circuitboard/machine/processing_unit_console
 	var/obj/machinery/mineral/processing_unit/machine = null
```

Comment:
When this is destroyed, it needs to set machine.CONSOLE to null in order to avoid hard-dels